_posts/2022-07-20-developer-cli.markdown

```diff
@@ -106,6 +106,16 @@ As things might not always be clear, here are some tips to help you:
 * Workloads are run under the _flotta_ user. Workloads debugging needs to be done also as _flotta_ user on the device.
 
 ### Debug the device
+For registering a device, the operator should download certificates and set them the correct permissions:
```

Comment:
"the operator" - isn't precise. it is the flotta-edge-api that creates the certificates.

"I'd phrase this as: "In order to register the device, the Flotta Edge API certificates need to be used to establish communication. To download the certificates run the following from flotta-operator project."

Reply:
done